api/routes/apds/get.test.js

```diff
@@ -80,6 +80,7 @@ tap.test('apds GET endpoint', async endpointTest => {
       await handler({ params: {}, user: { state: 'va' } }, res);
 
       validTest.ok(res.status.notCalled, 'HTTP status not explicitly set');
+      validTest.ok(ApdModel.where.calledWith({ state_id: 'va' }));
```

Comment:
Make this test a little more robust by verifying that the database fetch is constrained.